pilot/pkg/networking/core/cluster.go

```diff
@@ -235,7 +235,7 @@ func (configgen *ConfigGeneratorImpl) buildClusters(proxy *model.Proxy, req *mod
 		// Waypoint proxies do not need outbound clusters in most cases, unless we have a route pointing to something
```

Comment:
I guess this is not true anymore, given the tracer may need an outbound cluster.